lib/hls/hls_parser.js

```diff
@@ -2004,23 +2078,58 @@ shaka.hls.HlsParser = class {
     return null;
   }
 
+  /**
+   * Attempts to guess which codecs from the codecs list belong to a given
```

Comment:
I think this comment is copied from elsewhere.  It doesn't match this method.

Reply:
When copying a method I forgot to update the comment. It's already solved.